annot/src/main/java/com/predic8/membrane/annot/generator/HelpReference.java

```diff
@@ -64,13 +68,30 @@ public void writeHelp(Model m) {
 
 	}
 
-	private void writeFiles(Model m, String path) throws TransformerException {
+	private void writeFiles(Model m, String path) throws TransformerException, IOException {
 		// indent
 		TransformerFactory factory = TransformerFactory.newInstance();
 		Transformer transformer = factory.newTransformer();
 		transformer.setOutputProperty(OutputKeys.INDENT, "yes");
 		transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
-		transformer.transform(new StreamSource(new StringReader(sw.toString())), new StreamResult(new File(path + "/" + getFileName(m) + ".xml")));
+
+		String xml = sw.toString();
+		try {
+			transformer.transform(new StreamSource(new StringReader(xml)), new StreamResult(new File(path + "/" + getFileName(m) + ".xml")));
+		} catch (Exception e) {
+			log.error("**************************************************************************");
+			log.error("Error Parsing generated XML. " + e);
```

Comment:
https://stackoverflow.com/questions/48286809/how-to-correctly-produce-errors-from-a-javac-plugin